tests/test_trigger_bisect_jobs.py

```diff
@@ -203,6 +204,19 @@ def test_triggers():
     ]
     assert prio_calls == openqa.openqa_set_job_prio.call_args_list
 
+def test_catch_CalledProcessError():
+    from subprocess import CalledProcessError
+    args = args_factory()
+    args.url = "https://openqa.opensuse.org/tests/7848818"
+    openqa.fetch_url = MagicMock(side_effect=mocked_fetch_url)
+    openqa.openqa_clone = MagicMock(side_effect=CalledProcessError(returncode=255,
+                                                                   cmd='foo',
+                                                                   stderr="foo failed with return code 255"))
+    with pytest.raises(CalledProcessError) as expected:
+        openqa.main(args)
+    assert expected.value.returncode == 255
+    assert expected.value.cmd == 'foo'
+    assert "Command 'foo' returned non-zero exit status 255." in str(expected.value)
```

Comment:
This part I don't understand. It looks like you assert what should happen *before* your change but you want to avoid the stack trace so you should also assert that there is *no* traceback showing up, right?

Reply:
Sure I could. When I submitted, I didnt mean the end of the commit. I had to push a draft as you asked to provide one at the end of the day. I will work on it